src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -146,6 +146,20 @@ <h2 class="question-details"><b>Question {{ model.questionNumber }}: </b>{{ mode
           </div>
 
           <div id="comment-section-qn-{{ model.questionNumber }}-idx-{{ i }}" class="col-12 margin-bottom-20px margin-top-10px indent">
+            <div *ngIf="recipientSubmissionFormModel.commentByGiver">
+              <div class="card-body visibility-card">
+                <p class="text-secondary">The following persons can see your comments:</p>
```

Comment:
Minor nit. Let's standardise with the response block above.
```suggestion
                <p class="text-secondary">Only the following persons can see your comments: </p>
```